media/css/externalpages/pocket/components/_mobile-nav.scss

```diff
@@ -0,0 +1,148 @@
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at https://mozilla.org/MPL/2.0/.
+
+@import '~@mozilla-protocol/core/protocol/css/includes/lib';
+@import '../utils/variables';
+
+// variables from pocket
+$easing-accelerate: cubic-bezier(0.4, 0, 1, 1);
+$easing-decelerate: cubic-bezier(0, 0, 0.2, 1);
+
+.mobile-nav-wrapper {
+    background: rgba(26, 26, 26, 0.4) none repeat scroll 0 0;
+    bottom: 0;
+    display: none;
+    left: 0;
+    mix-blend-mode: normal;
+    opacity: 0;
+    pointer-events: none;
+    position: fixed;
+    right: 0;
+    transition: opacity 150ms $easing-accelerate, transform 75ms $easing-accelerate;
+    top: 0;
+    z-index: 20;
+
+    &.active {
+        display: block;
+        pointer-events: auto;
+    }
+}
+
+.mobile-nav {
+    background: $color-white;
+    border-radius: 0;
+    bottom: 0;
+    box-shadow: 0 4px 12px rgba(0, 0, 0, 0.12);
+    left: 0;
+    max-height: 100%;
+    max-width: 256px;
+    position: absolute;
```

Comment:
When testing on my iPhone I notice the list is not accessible if the items within it are taller than the screen. For example, if I have my phone in landscape mode then I can't access the items toward the bottom of the list.

When testing the mobile menu on the Pocket site in prod, I seem to be able to scroll the list ok?

Reply:
ok, let me take a look - I totally forgot to check landscape mode.

Reply:
What you have now works great 👍